app/assets/stylesheets/users.css.scss

```diff
@@ -0,0 +1,19 @@
+#img_prev {
+  width: 200px;
+  height: 200px;
+  object-fit: cover;
+  object-position: center;
+  border-radius: 50%;
+  z-index: 0;
+}
```

Comment:
Is it possible to use tailwind css for css? 
This was also mentioned in the development documentation.